portfolio/src/main/webapp/index.html

```diff
@@ -1,16 +1,26 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-    </div>
-  </body>
-</html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>My Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<div id="fullpage">
```

Comment:
This PR appears to be doing two independent things: removing the old HTML content *and* adding fullpage functionality. Consider splitting those up or applying just one of those things. Also, I suggest you don't leave all those empty sections -- think about what you wanna put in there; maybe use some placeholders, or consider adding [TODOs](https://google.github.io/styleguide/htmlcssguide.html#Action_Items).

Reply:
Thanks, I'll push a commit through soon that adds some TODOs.

Reply:
I'll also consider how to split up the PR.

Reply:
@krasimir-google, I've gone ahead and reverted the commit 5e68066 and have added back the boilerplate content. I will create a separate pull request to remove this content.